internal/k8s/controller.go

```diff
@@ -219,7 +219,7 @@ func NewLoadBalancerController(input NewLoadBalancerControllerInput) *LoadBalanc
 		}
 	}
 
-	if input.ReportIngressStatus && input.IsLeaderElectionEnabled {
+	if input.IsLeaderElectionEnabled {
```

Comment:
@pleshakov note this change, in case you think this can cause any issue

Reply:
good catch 👍 seems like we had a bug